usr/share/rear/lib/uefi-functions.sh

```diff
@@ -46,11 +46,11 @@ function build_bootx86_efi {
         Log "Did not find grub-mkimage (cannot build bootx86.efi)"
         return
     fi
-    if [[ -f /etc/slackware-version ]] || [[ -f /etc/fedora-release ]] ; then
-        # slackware grub doesn't have linuxefi module
-        $gmkimage $v -O x86_64-efi -c $TMP_DIR/mnt/EFI/BOOT/embedded_grub.cfg -o $TMP_DIR/mnt/EFI/BOOT/BOOTX64.efi -p "/EFI/BOOT" part_gpt part_msdos fat ext2 normal chain boot configfile linux multiboot jfs iso9660 usb usbms usb_keyboard video udf ntfs all_video gzio efi_gop reboot search test echo btrfs
-    else
-        $gmkimage $v -O x86_64-efi -c $TMP_DIR/mnt/EFI/BOOT/embedded_grub.cfg -o $TMP_DIR/mnt/EFI/BOOT/BOOTX64.efi -p "/EFI/BOOT" part_gpt part_msdos fat ext2 normal chain boot configfile linux linuxefi multiboot jfs iso9660 usb usbms usb_keyboard video udf ntfs all_video gzio efi_gop reboot search test echo btrfs
-    fi
+    # as not all Linux distro's have the same kernel modules present we verify what we have (see also https://github.com/rear/rear/pull/2001)
```

Comment:
I think `kernel modules` is a typo which actually should be `Grub modules`
at least as far as I understand things e.g. according to
https://blog.fpmurphy.com/2010/06/grub2-modules.html
that reads (excerpt)
```
GRUB2 has a modular architecture with support
for loading and unloading code modules as needed.
Modules are similar to Linux kernel modules in some respects
but in other ways they are like plugins for Mozilla Firefox
```